pkg/kubelet/config/etcd.go

```diff
@@ -61,6 +61,10 @@ func (s *sourceEtcd) run() {
 	boundPods := api.BoundPods{}
 	err := s.helper.ExtractObj(s.key, &boundPods, false)
 	if err != nil {
+		if (tools.IsEtcdNotFound(err)) {
+			glog.V(4).Infof("etcd failed to retrieve the value for the key %q. Error: %v", s.key, err)
```

Comment:
Might a more specific error message be appropriate / more helpful here? Does this mean it can't talk to etcd at all?